pkg/tfgen/generate_nodejs_utilities.go

```diff
@@ -49,4 +49,8 @@ export function requireWithDefault<T>(req: () => T, def: T | undefined): T {
     }
     return def;
 }
+
+export function getVersion(): string {
+    return require('./package.json').version.slice(1);
```

Comment:
I think the intention of this slice is to remove the leading `v` from the version, correct?

`package.json` does allow for you to elide the leading `v`, and while all our generated packages will have this as an artifact of how we do version tagging, do you think we could augment this to not to the trim if the v isn't present?

Reply:
> package.json does allow for you to elide the leading v

Ah, I did not know that - I think you're right, we should do that.